### DISCUSSION TITLE
Batch Subtitle Embed/Burn for Multiple Video files

### DISCUSSION THREAD

**Discussion by Dattebayo2505** [SubtitleEdit/subtitleedit#7906] [Ideas]
Is it possible to batch embed multiple subtitles onto their respective video files? The current option is through Video -> Generate video with...
However, this is only for single files. Is there a way that let's the program auto-detect the names of the subtitles automatically (especially since after doing audio-to-text - the name of the generated subtitle files is the same with the video name)? I may be able to use other programs to do this task; however, SubtitleEdit is already capable of (seamlessly) converting a video to a different format whilst adding subtitles (which is such a nice feature).

OR, I am just dumb and there is a way(?) to do so already?

Also, someone else mentioned this [last year](https://github.com/SubtitleEdit/subtitleedit/issues/5897).

Thank you!

**Comment by Dattebayo2505** [comment]
Added 02/09/2024:

I have searched, and fortunately found an app that does batch processes [here](https://github.com/yaser01/mkv-muxing-batch-gui/releases). Credits to [yaser01](https://github.com/yaser01).

**Comment by sherkhan30452** [comment]
It has been officially added to subtitle edit 4.0.6 see [changelog](https://raw.githubusercontent.com/SubtitleEdit/subtitleedit/master/Changelog.txt)

**Reply by Dattebayo2505** [reply]
Thank you very much for this info!
